src/inquirer.ts

```diff
@@ -18,8 +18,8 @@ export const functionNamePrompt = (source: functionNameSource) => {
     message: 'Select a functionName',
     type: 'autocomplete',
     source,
-  };
-  return prompt<{ functionName: string }>([question]);
+  } as unknown as Question;
```

Comment:
Can't have `as unknown`.

Reply:
you cannot directly cast the object with autocomplete specific attributes into inquirer's `Question` type. the ` as unknown as <type>` is the only way I could find (in Typescript issues)

Reply:
OK, we can fix this later.

Reply:
Best `fix` would require to contribute to `@types/inquirer` so that the type `Question` accept extra attributes.
Ideally, the autocomplete plugin should be typed too. That project does not look very active though.